bcManager/bcManager.py

```diff
@@ -587,16 +601,26 @@ async def bcgroup(self, ctx):
         else:
             await ctx.send(":x: A ballchasing group has not been set for this season.")
 
+    @commands.command()
+    @checks.is_owner()
+    async def console(self, ctx):
```

Comment:
maybe rename as `clear_console`? `console` isn't clear what it does

Reply:
Renamed and added aliases.